forward_grpc_test.go

```diff
@@ -224,7 +224,8 @@ func TestE2EForwardingGRPCMetrics(t *testing.T) {
 		}
 
 		assert.ElementsMatch(t, expectedNames, actualNames,
-			"The global Veneur didn't flush the right metrics")
```

Comment:
meep? Did something happen here?

Reply:
Yeah, this test seemed flaky so I wanted to debug it, and now it passes...